test/testuser.py

```diff
@@ -81,6 +81,42 @@ def cannot_see_any_recordings(self):
                 "User '{}' can see a recording from '{}'".format(
                     self.username, recordings[0]['Device']['devicename']))
 
+    def can_download_correct_recording(self, recording):
+        content = io.BytesIO()
```

Comment:
This is a very 'programic way to test this'   So it tests this but it 
a) doesn't clarify very well what is tested
b) and would be easy to get the two props arrays confused so you get a test that can't fail.  
It's fine, but maybe we should talk about it on Thursday.

Reply:
I quite like having a test helper which checks that a recording can be downloaded and has the correct metadata. Happy to discuss other approaches on Thursday (putting it on my list).